aux/gv_results.slim

```diff
@@ -33,16 +33,12 @@ html
             h4.modal-title Incompatible Browser!
```

Comment:
Maybe 'Some features disabled'. Or maybe 'Viewing graphs disabled', if this is the only feature disabled.